src/components/Table/readme.md

```diff
@@ -981,6 +981,145 @@ function TableExample(props) {
 <TableExample data={data} />;
 ```
 
+##### Data Table with custom actions and wrap description
+
+```js
+import {
+    Table,
+    Column,
+    ButtonGroup,
+    ButtonIcon,
+} from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styled from 'styled-components';
+import React, { useState } from 'react';
+import {faEllipsisV,
+        faPlus,
+        faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+
+const data = [
+    {
+        name: 'Leandro Torres',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '1234qwerty',
+    },
+    {
+        name: 'Jose Torres',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '1234asdfgh',
+    },
+    {
+        name: 'Reinier',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '1234zxcvbn',
+    },
+    {
+        name: 'Sara',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '5678qwerty',
+    },
+    {
+        name: 'Tahimi',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '5678asdfgh',
+    },
+    {
+        name: 'Leo',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '5678zxcvbn',
+    },
+    {
+        name: 'Tahimi Leon',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '9012qwerty',
+    },
+    {
+        name: 'Alejandro',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',  
+        id: '5678zdfgdf',
+    },
+    {
+        name: 'Carlos',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '3434fgfgdf',
+    },
+    {
+        name: 'Luis',
+        description:'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et.',
+        id: '9012asdfgh',
+    },
+];
+const Title = styled.h1.attrs(props => props.theme.rainbow)`        
+    font-family: 'Lato Light';
+    font-size: 25px;
+    text-align: left;
+    margin-bottom: 10px;
+    color: ${props => props.palette.text.main};
+    `;  
+
+const Subtitle = styled.h3.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.title};
+    `;
+
+function TableCustomAction(props) {
+    const [data, setData] = useState(props.data);
```

Comment:
Copying props to a state is bad practice. I think a cleaner solution would be this:

```js
 const [data, setData] = useState(initialData);
```

and remove the prop in the component call.